pkgs/servers/home-assistant/tests.nix

```diff
@@ -78,6 +78,12 @@ let
   };
 
   extraPytestFlagsArray = {
+    backup = [
+      "--snapshot-update"
```

Comment:
I don't think that's a good idea. Rather disable the one test that's failing.

Reply:
Removed for backup and bmw_connected_drive, matter still needs work.